### PR TITLE
Gulpfile fix

### PR DESCRIPTION
**Pull request by krshubham**
This Pull Request solves the following issue:

[Gulp process stops with javascript files with syntax errors](https://github.com/Grasia/teem/issues/4)

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -371,95 +389,120 @@ gulp.task('html:production', function() {
...
+      .on('error', endErrorProcess),
     // rest of app logic
     gulp.src(['./src/js/**/*.js', '!./src/js/app.js', '!./src/js/widgets.js'])
-    .pipe(sourcemaps.init())
-    .pipe(babel({
-      presets: ['es2015'],
-      plugins: ['transform-object-assign']
-    }))
-    .pipe(ngFilesort()),
+      .pipe(sourcemaps.init())
```
Comment (atfornes) [inline gulpfile.js]: Changing the indentation of not changed code might bring issues when trying to track changes in git history. Could you please provide a pull request where only the changed code appears as change? Thanks!

Please note that this happens through most of the file changes of the pull request.

Comment (krshubham) [reply gulpfile.js]: I am sorry about that! My editor automatically takes the jscs and jshint configs and makes all the code follow the rule! Sorry for this! I'll take care in future

Comment (atfornes) [reply gulpfile.js]: we are lucky and we can use  `git merge -Xignore-space-change` as proposed [here](https://stackoverflow.com/a/9784089/4928558).

Comment (krshubham) [reply gulpfile.js]: Okay I'll fix this right away

Comment (atfornes) [reply gulpfile.js]: I finally had to use a bit more complex approach, shared in [this gist](https://gist.github.com/atfornes/f4a917c6c0cf5b773824a6545ae2ef5e)

**Comment by atfornes** [comment]
merged at https://github.com/Grasia/teem/commit/4f6431b3cc6956b40bd1c2f9b7d6ef4eaf22dfc8

**Pull request closed without merging**